src/dune_rules/melange/melange_rules.ml

```diff
@@ -236,9 +245,7 @@ let setup_emit_cmj_rules ~sctx ~dir ~scope ~expander ~dir_contents
         ()
       in
       match mel.alias with
-      | None ->
-        let alias = Alias.make Melange_stanzas.Emit.implicit_alias ~dir in
-        Rules.Produce.Alias.add_deps alias emit_and_libs_deps
+      | None -> add_deps_to_default_ailases ~dir emit_and_libs_deps
       | Some alias_name ->
```

Comment:
We should also add the JS rules to the default alias, even if `melange.emit` has `(alias foo)`.

Reply:
gotcha

Reply:
fixed in 72202babe